servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/Single.java

```diff
@@ -2599,10 +2575,20 @@ void handleSubscribe(Subscriber<? super T> subscriber, SignalOffloader signalOff
     /**
      * Returns the {@link Executor} used for this {@link Single}.
      *
-     * @return {@link Executor} used for this {@link Single} via {@link #Single(Executor)}.
+     * @return {@link Executor} used for this {@link Single}.
+     */
+    Executor executor() {
```

Comment:
inline with above comments, this will eventually go away too IIUC?

Reply:
Yes. once the SignalOffloader creation is removed from `subscribeWithContext` this can go away.